topi/python/topi/arm_cpu/conv2d.py

```diff
@@ -700,6 +701,9 @@ def _alter_conv2d_layout_arm(attrs, inputs, tinfos, F):
 
     new_attrs = {k: attrs[k] for k in attrs.keys()}
 
+    if F == tvm.relay.op:
+        new_attrs["channels"] = infer_channels(inputs[1], False)
```

Comment:
Should use `attrs['kernel_layout"]` to support all layouts

Reply:
@merrymercy,
Fixed now.